app/views/projects/show.html.erb

```diff
@@ -0,0 +1,4 @@
+<h1><%= @project.name %></h1>
+<p>Material: <%= @project.material%></p>
+<p>Theme: <%= @theme %></p>
+<p>Number of Contestants: <%= @project.contestants.count%>
```

Comment:
This is a minor MVC infraction (and we allow 1 infraction as part of the rubric of the challenge) just because `count` is a calculation method so we want to keep it close to the data/model. It'd be better to encapsulate this functionality in a project model method called `count_contestants` or something and then call this method here rather than calling `count` in the view